addons/portal_rating/static/src/js/portal_chatter.js

```diff
@@ -346,13 +346,14 @@ PortalChatter.include({
         var messageIndex = $source.data("mes_index");
 
         var comment = this.messages[messageIndex].rating.publisher_comment;
-        this._getCommentContainer($source).empty();
```

Comment:
Wouldn't it be a bit cleaner to store the comment container into a variable and then use it to do the operations?

So:

```
const $commentContainer = this._getCommentContainer($source);
$commentContainer.empty();
if (comment) {
    ...
    $commentContainer.append(...);
}
```